product_docs/docs/tpa/23/reference/volumes.mdx

```diff
@@ -28,35 +28,37 @@ instances:
   - device_name: root
     volume_type: gp2
     volume_size: 32
-  - raid_device: /dev/md0
-    device_name: /dev/xvdf
+  - device_name: /dev/xvdf
     volume_type: io2
     volume_size: 64
-    raid_units: 2
-    raid_level: 1
     iops: 5000
     vars:
       volume_for: postgres_data
       encryption: luks
-  - raid_device: /dev/md1
-    device_name: /dev/xvdh
+  - device_name: /dev/xvdb
     ephemeral: ephemeral0
-    raid_units: all
     vars:
       mountpoint: /mnt/scratch
 ```
 
 In this example, the EC2 instance will end up with a 32GB EBS root
-volume, a 64GB RAID-1 volume comprising two provisioned-iops EBS volumes
-mounted as /opt/postgres/data, and a /tmp/scratch filesystem comprising
-all available instance-store (“ephemeral”) volumes, whose number and
-size are determined by the instance type.
+volume, a 64GB io2 volume (provisioned-iops EBS volumes) mounted as
+/opt/postgres/data, and a /tmp/scratch filesystem provided by an
+instance-store (“ephemeral”) volume, whose number and size are determined
+by the instance type.
 
 The details are documented in the section on AWS below, but settings
 like `volume_type` and `volume_size` are used during provisioning, while
 settings under `vars` like `volume_for` or `mountpoint` are written to
 the inventory for use during deployment.
 
+!!! NOTE ephemeral0 instance store
```

Comment:
Here too, might be: 
```suggestion
!!!note NOTE ephemeral0 instance store
```
and the text below indented respectively.

Reply:
Will fix upstream in next pass.